src/app/cic/steps.js

```diff
@@ -239,7 +239,7 @@ module.exports = {
     editBackStep: "checkDetails",
     next: [
       {
-        field: "nonUKPassportExpiryDate",
+        field: "euPhotocardDlExpiryDate",
```

Comment:
Is there a unit test that covers this flow?

Reply:
It's my understanding that this doesn't need unit testing as it's passing everything over to the framework. This logic used to be in our controllers (which have unit tests), but was since refactored to use the framework correctly and take away our own (and redundant) routing logic